Formula/v/vulture.rb

```diff
@@ -16,16 +18,10 @@ class Vulture < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "63f6b4f4c33053e4f2e22fe9f090066beb906de180259ddec58ee6819aa7bbf5"
   end
 
-  depends_on "python-setuptools" => :build
-  depends_on "python-toml"
```

Comment:
Upstream no longer requires separate toml package for Python >=3.12, uses `tomllib` from the stdlib: https://github.com/Homebrew/homebrew-core/pull/new/vulture-vendor-deps